CHANGELOG.md

```diff
@@ -13,13 +13,21 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 ### Fixed
 
-- []() Store: Improved synchronization of meta JSON files. Store now properly handles corrupted disk cache. Added meta.json sync metrics.
+- [issue#1919](https://github.com/thanos-io/thanos/issues/1919) Compactor: Fixed potential data loss when uploading older blocks, or upload taking long time while compactor is 
```

Comment:
```suggestion
- [#1919](https://github.com/thanos-io/thanos/issues/1919) Compactor: Fixed potential data loss when uploading older blocks, or upload taking long time while compactor is 
```

Reply:
It's an issue so wanted to make sure it's verbose ;p but happy to change